src/api/gpt/index.ts

```diff
@@ -0,0 +1,123 @@
+import OpenAI from "openai";
+import env from "../../constants/env";
+
+const openai = new OpenAI({
+  apiKey: env.OPENAI_API_KEY,
+  dangerouslyAllowBrowser: true,
```

Comment:
add a `TODO` to move the open ai to server-side. Yet we may not need to address it for now.